dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
```

Comment:
We should probably start off with some context on what a configuration is / its relationship with a template. Maybe at least move the later line "When OpenShift processes a template, it creates a configuration" up to here to start this para off.

For the current sentence on this line, it's maybe also more accurate to say something like: "You can create a configuration from a template using the CLI or, if templates have been uploaded to your project or global template library, using the Management Console."